boot/boot.go

```diff
@@ -442,6 +451,37 @@ func makeBootable20RunMode(model *asserts.Model, rootdir string, bootWith *Boota
 		return fmt.Errorf("cannot set recovery system environment: %v", err)
 	}
 
+	// get the ubuntu-boot grub and extrac the kernel there
+	opts = &bootloader.Options{
+		// TODO:UC20: we use "recovery: true" here because on
+		// the partition the file layout of ubuntu-boot looks
+		// the same as ubuntu-seed
+		Recovery:              true,
```

Comment:
this is confusing name now then. I need to look how it's used to propose a different one, but not a blocker.